routers/frontend.js

```diff
@@ -6,8 +6,23 @@ frontendRouter.get("/", (_, res) => {
   res.sendFile(appRoot + "/public/index.html");
 });
 
+
+frontendRouter.get("/random", (_, res) => {
+  res.sendFile(appRoot + "/public/pages/randomIdiom.html");
+});
+
+frontendRouter.get("/mainCSS", (_, res) => {
+  res.sendFile(appRoot + "/public/styles/styles.css");
```

Comment:
We already have routes to serve our js and css. You just need to set the href to `/styles/styles.css` and `/js/randomIdiom.js`

Reply:
Will make the change, thanks

Reply:
Done